misc/python/materialize/checks/all_checks.py

```diff
@@ -45,7 +44,6 @@
 from materialize.checks.rename_table import *  # noqa: F401 F403
 from materialize.checks.rename_view import *  # noqa: F401 F403
 from materialize.checks.replica import *  # noqa: F401 F403
-from materialize.checks.roles import *  # noqa: F401 F403
```

Comment:
Why is this getting removed?

Reply:
This should be a draft PR; sorry for the confusion